onadata/apps/logger/models/instance.py

```diff
@@ -79,6 +79,9 @@ def _get_attachments_from_instance(instance):
         attachment['id'] = a.id
         attachments.append(attachment)
 
+        if isinstance(a.deleted_at, datetime):
```

Comment:
Since `deleted_at` defaults to null, couldn't we just change this to `if a.deleted_at` ?